lib/rototiller/task/params/switch.rb

```diff
@@ -0,0 +1,77 @@
+require 'rototiller/task/collections/env_collection'
+require 'rototiller/task/block_handling'
+
+module Rototiller
+  module Task
+
+    # The Switch class to implement rototiller command switch handling
+    #   via a RototillerTask's #add_command and Command's #add_switch
+    #   contains information about a Switch's state, as influenced by environment variables, for instance
+    # @since v1.0.0
+    # @attr [String] name The name of the switch to add to a command string
+    class Switch < RototillerParam
```

Comment:
Does switch not have a message?  I guess messages are only important when collecting input from the ENV?